docs/source/spec-workload-executor.rst

```diff
@@ -37,55 +37,133 @@ Behavioral Description
 
 After accepting the inputs, the workload executor:
 
-#. MUST use the input connection string to instantiate the ``MongoClient`` of the driver that is to be tested.
-   Note that the workload executor:
+#. MUST use the input connection string to `instantiate the
+   unified test runner <https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#id92>`_
+   of the driver being tested. Note that the workload executor:
 
    * MUST NOT override any of the URI options specified in the incoming connection string.
    * MUST NOT augment the incoming connection string with any additional URI options.
```

Comment:
Is this still true? Please see my question above about how we expect to honor URI options.